src/clj_config/core.clj

```diff
@@ -183,4 +203,14 @@
 
   (defconfig
     :app {sentry-url :sentry-dsn}
```

Comment:
the hash-maps are no longer supported. Should this be updated to the vector version
[sentry-url :sentry-dsn]